modules/browser/capabilities.json

```diff
@@ -0,0 +1,121 @@
+{
+    "safari": [
+        {"version": "6.3" },
+        {
+            "version": "10.2",
+            "capabilities": {
+                "audioIn": true,
+                "audioOut": true,
+                "videoIn": true,
+                "videoOut": true,
+                "screenSharing": true
+            }
+        },
+        {
+            "capabilities": {
+                "audioIn": true,
+                "audioOut": true,
+                "videoIn": false,
+                "videoOut": false,
+                "screenSharing": false
+            },
+            "iframeCapabilities": {
+                "isSupported": false
+            }
+        }
+    ],
+    "chrome":
+    [
+        {"version": "34" },
+        {
+            "capabilities": {
+                "audioIn": true,
+                "audioOut": true,
+                "videoIn": true,
+                "videoOut": true,
+                "screenSharing": true
+            }
+        }
+    ],
+    "opera":
+    [
+        {"version": "22" },
+        {
+            "capabilities": {
+                "audioIn": true,
+                "audioOut": true,
+                "videoIn": true,
+                "videoOut": true,
+                "screenSharing": false
+            }
+        }
+    ],
+    "firefox": [
+        {"version": "52.4" },
+        {
+            "capabilities": {
+                "audioIn": true,
+                "audioOut": true,
+                "videoIn": true,
+                "videoOut": true,
+                "screenSharing": true
+            }
+        }
+    ],
+    "iexplorer": [
+        {"version": "9" },
+        {
+            "capabilities": {
+                "audioIn": true,
+                "audioOut": true,
+                "videoIn": true,
+                "videoOut": true,
+                "screenSharing": true
+            }
+        }
+    ],
+    "edge": [
+        {
+            "capabilities": {
+                "audioIn": true,
+                "audioOut": true,
+                "videoIn": true,
+                "videoOut": true,
+                "screenSharing": false
+            }
+        }
+    ],
+    "nwjs": [
+        {
+            "capabilities": {
+                "audioIn": true,
+                "audioOut": true,
+                "videoIn": true,
+                "videoOut": true,
+                "screenSharing": false
+            }
+        }
+    ],
+    "electron": [
+        {
+            "capabilities": {
+                "audioIn": true,
+                "audioOut": true,
+                "videoIn": true,
+                "videoOut": true,
+                "screenSharing": false
```

Comment:
Why is screensharing false for electron?

Reply:
Good catch! Copy/paste mistake :)